packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -772,28 +773,7 @@ export const DatePickerMixin = (subclass) =>
 
     /** @private */
     _formatISO(date) {
-      if (!(date instanceof Date)) {
```

Comment:
Extracted this to the helpers, so that the two versions of this functions are easier to correlate.